frontend/src/utils/navbar.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
```

Comment:
The file has moved, do recreate it at its former place

Reply:
Checked again with master branch and find